scripts/lp-create-bug

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+"""Create bugs in launchpad for the specified project"""
+
+import argparse
+import sys
+import re
+from launchpadlib.launchpad import Launchpad
+
+BUG_IMPORTANCE = ['Low', 'Wishlist', 'Medium', 'High', 'Critical']
+BUG_STATUS = ['New', 'Confirmed', 'Triaged', 'In Progress', 'Fix Committed']
+CREATE_BUG_MESSAGE_TMPL = """
+Creating a bug in {project}:
+Importance: {importance}
+Status:     {status}
+Tags:       {tags}
+Title: {title}
+
+{description}
+"""
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__)
+
+    parser.add_argument('--title', required=True, action='store',
+                        help="Bug title string")
+    parser.add_argument('--tags', action='append', default=[],
```

Comment:
I would name the option `--tag`, as it accepts only one tag, and setting multiple tags requires to specify the option multiple times.

Reply:
Thanks again, done. and fixed flakes